build-tools/packages/build-cli/src/commands/modify/lockfile.ts

```diff
@@ -0,0 +1,96 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { updatePackageJsonFile } from "@fluidframework/build-tools";
+import { Flags } from "@oclif/core";
+import execa from "execa";
+
+import { BaseCommand } from "../../base";
+import { releaseGroupFlag } from "../../flags";
+
+/**
+ * Updates the version of a dependency in the lockfile.
+ *
+ * @remarks
+ * This command is primarily used manually to update transitive dependencies when we need to address CVEs.
+ *
+ * Note that this applies to all packages in the specified release group.
+ */
+export default class UpdateDependencyInLockfileCommand extends BaseCommand<
+	typeof UpdateDependencyInLockfileCommand
+> {
+	static readonly summary =
+		"Updates a dependency in the lockfile to the latest version of a specified semver range.";
+
+	static readonly description =
+		`Note that if the version passed in to the command is not within the range of versions ` +
+		`naturally accepted by the packages that depend on it, after this command runs the lockfile might not reflect the ` +
+		`version that was passed in, but the latest version that complies with the semver range declared by the dependent packages.`;
+
+	static readonly enableJsonFlag = true;
+
+	static readonly flags = {
+		releaseGroup: releaseGroupFlag({ required: true }),
+		dependencyName: Flags.string({
+			description: "Name of the dependency (npm package) to update",
+			required: true,
+		}),
+		version: Flags.string({
```

Comment:
Note for the future - you can have a parse property that's a function that receives the value the user rpovided, and returns the value the code should use. That lets you verify, for example, that the user-provided string is a valid semver at flag parsing time.